info-commands.go

```diff
@@ -525,6 +526,7 @@ type Disk struct {
 	Utilization     float64      `json:"utilization,omitempty"`
 	Metrics         *DiskMetrics `json:"metrics,omitempty"`
 	HealInfo        *HealingDisk `json:"heal_info,omitempty"`
+	OfflineInfo     *OfflineInfo `json:"offline_info,omitempty"`
```

Comment:
You couldn't know that, but I am trying to reduce the load of DiskInfo, and remove costly stats from it, since it is passed around so much.

Therefore I don't particularly like passing around an un-bounded array here, since you can get thousands of drives easily.

So I'd prefer not to add *more* to this, but don't really have a solid alternative.

Reply:
@klauspost Yeah there is a lot here, one potential solution is to have `ec` make more API requests to hydrate / populate the offline tracker information that it needs for `mc admin heal -v`, but that is for sure slower on the client side. Would you prefer that approach?

Reply:
@klauspost Are you ok with leaving OfflineInfo as a struct member of madmin.Disk, and then we only populate it when the handler being called is for `mc admin heal -v`? It's `/minio/admin/v4/background-heal/status`. 

It's a pointer like other things, so needs a nil check before use, and it has `omitempty` so in JSON won't be visible if empty.

The commit that implements this is here: https://github.com/miniohq/eos/pull/1301/commits/721656f68034fe14d4117263b28bfa34b29a26bc

Reply:
I chatted to Klaus in DM on Slack and he seems ok with this approach